Rlog/View/Components/CustomTextField.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  CustomTextField.swift
+//  MacCPlayGround
```

Comment:
```suggestion
//  Rlog
```

Project name과 동일한 이름으로 변경되면 좋을 것 같아요 ✨

Reply:
[b40f63f](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Bremen-DD/pull/10/commits/b40f63fb32e3005219c31bbf87e8badf704a31c2) 수정했습니다!